submissions/oluwatosinOke/database/init.sql

```diff
@@ -0,0 +1,135 @@
+('iphone-11', 'Powered by 3110 mAh non-removable battery. Runs iOS 13 and equipped with A13 Bionic chip. 4GB of RAM + 64GB internal storage; 4GB of RAM + 128GB internal storage; 4GB of RAM + 256GB internal storage.', '../html/images/iPhone-11.jpg', 1, 1, 120000, 2, 'LOW STOCK'),
+('Nokia 6.2', 'Processor: Qualcomm SDM636 Snapdragon 636 Octa-core 1.8 GHz Kryo 260 · RAM: 3 GB, 4 GB · Storage: 32 GB, 64 GB, 128 GB · Display: 6.3 inches.', '../html/images//Nokia-6.2.jpg', 1, 2, 90000, 0, 'OUT OF STOCK'),
+('Infinix Zero 8', 'TOS: Android 10 with XOS 7. Display: 6.85" full HD+ with 90Hz refresh rate and 180Hz touch sampling rate. CPU: Octa-core MediaTek Helio G90T SoC. RAM: 8 GB. Internal storage: 128 GB. Battery: 4,500 mAh', '../html/images/Infinix-Zero-8.jpg', 1, 2, 10000, 14, 'IN STOCK');
+
+-- Create table named customers
+CREATE TABLE customers (
+id INT NOT NULL AUTO_INCREMENT,
+first_name VARCHAR(30) NOT NULL,
+last_name VARCHAR(30) NOT NULL,
+email_address VARCHAR(30) NOT NULL,
+password VARCHAR(30) NOT NULL,
+PRIMARY KEY (id)
+);
+
+-- INSERT values into Table named customers
+INSERT INTO customers (first_name, last_name, email_address, PASSWORD)
+VALUES ('Aliko', 'Dangote', 'alikodangote@gmail.com', 'spagetti'),
+('Meghan', 'Markle', 'merganmarkle@gmail.comn', 'princess'),
+('Kim', 'Kardasian', 'kimk@gmail.comn', 'northwest'),
+('David', 'Adeleke','davidomusic@gmail.com', 'goodtime');
+
+
+-- CREATE TABLE named customers_address
+CREATE TABLE customers_address (
+id INT NOT NULL AUTO_INCREMENT,
+customer_id INT NOT NULL,
+street_address VARCHAR(150) NOT NULL,
+city VARCHAR(30) NOT NULL,
+state VARCHAR(30) NOT NULL,
+zip_code INT,
+country VARCHAR(30) NOT NULL,
+phone_number VARCHAR(30) NOT NULL,
+PRIMARY KEY (id),
+FOREIGN KEY (customer_id) REFERENCES customers (id)
+);
+
+-- INSERT values into Table named customers_address
+INSERT INTO customers_address (customer_id, street_address, city , state, zip_code, country, phone_number)
+VALUES (1, 'Dangote close', 'Ikeja', 'Lagos', 2020, 'Nigeria', '2347676'),
+(2, '13th Street. 47 W 13th Street', 'New York', 'NY', 10011, 'USA', '08132420'),
+(3, '20 Cooper Square', 'NEW YORK', 'NY', 10003, 'USA', '012983772'),
+(4, 'plot 302 Asokoro Extention', 'Abuja', 'FCT', 2090, 'Nigeria', '070373626');
+
+
+-- CREATE TABLE named orders
+CREATE TABLE orders (
+id INT NOT NULL AUTO_INCREMENT,
+customer_orders INT NOT NULL,
+order_amount DECIMAL(9.0) NOT NULL,
+ordercreated_at DATETIME,
+PRIMARY KEY (id),
+FOREIGN KEY (customer_orders) REFERENCES customers (id)
+);
+
+-- INSERT values into Table named orders
+INSERT INTO orders (customer_orders, order_amount, ordercreated_at)
+VALUES (1, 400000, NOW()),
+(2, 90000, NOW()),
+(3, 240000, NOW()),
+(4, 750000, NOW());
+
+
+-- CREATE TABLE named other_items
+CREATE TABLE other_items(
+id INT NOT NULL AUTO_INCREMENT,
+order_id INT NOT NULL,
+product_id INT NOT NULL,
+quantity INT NOT NULL,
+unit_price DECIMAL(9.0) NOT NULL,
```

Comment:
This is wrong for DECIMAL

Do you mean `DECIMAL(9, 0)`. The second value signifies, how many values after `.`. It will be best if you use `2` i.e `DECIMAL(9, 2)`